docs/datasets.md

```diff
@@ -55,3 +55,159 @@ To use `ROMS-Tools`, the user is required to download the following datasets:
 |  | -                      | `ratio_m2s`              | Ratio of volume between river mouth and station                                                  |  |      |
 |  | -                      | `riv_name`              | River name                                                  |  |      |
 |  | -                      | `vol_stn` (optional)             | Annual volume at station                                                  |  |      |
+
+# Downloading ERA5 and GLORYS
+
+## Prerequisites
+
+Install 2 packages into your conda python environment:
+
+**Copernicusmarine**: https://pypi.org/project/copernicusmarine/<br />
+**cdsapi**: https://anaconda.org/conda-forge/cdsapi
+
+## GLORYS
+
+### Setup
+
+Instructions on installation and how to set up account credentials, in the shell: [here](https://help.marine.copernicus.eu/en/articles/7970514-copernicus-marine-toolbox-installation)<br />
+But, essentially do:
+- 1st need to make an account with marine.copernicus: https://marine.copernicus.eu/
+- Export account credentials in `~/.bashrc`:
+- Linux:
```

Comment:
```suggestion
See the instructions [here](https://help.marine.copernicus.eu/en/articles/7970514-copernicus-marine-toolbox-installation). The `TLDR;` is:

- Install the Copernicus Marine Toolbox in your environment
- Make an account with [Copernicus Marine](https://marine.copernicus.eu/)
- Configure environment variables for your shell (e.g. `~/.bashrc`):

```

- Removed rogue comma
- Removed unnecessary colon in 1st bullet point
- Avoid verbosity of "but, essentially"
- Removed redundant "1st " (since it's the first bullet point).
    - Removed "need to ..." from "need to make ..."
- Added working link to marine.copernicus.eu
- Fixed incorrect naming of "Copernicus Marine" (was marine.copernicus)
- Removed "linux" bullet point since we don't duplicate other platforms